vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewSortingIT.java

```diff
@@ -116,6 +116,24 @@ public void gridWithSorting_invertAndResetSortings() {
 
     }
 
+    @Test
+    public void gridWithSorting_toggleColumnVisibilityAndReset() {
```

Comment:
The change itself works well, but the test case passes even without the fix in place

Reply:
Thanks, added the check for the actual `direction` attribute on the `vaadin-grid-sorter` element.